src/main/resources/config/liquibase/changelog/20220309161633_changelog.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="filipgregurevic" id="20220309161633-1">
+        <createTable tableName="plagiarism_case">
+            <column name="id" type="BIGINT" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" primaryKeyName="plagiarism_casePK"/>
+            </column>
+            <column name="exercise_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="student_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="post_id" type="BIGINT"/>
+            <column name="created_by" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_date" defaultValueComputed="CURRENT_TIMESTAMP" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_modified_by" type="VARCHAR(50)"/>
+            <column name="last_modified_date" type="TIMESTAMP"/>
+            <column name="verdict" type="VARCHAR(255)"/>
+            <column name="verdict_by_id" type="VARCHAR(255)"/>
+            <column name="verdict_date" type="TIMESTAMP"/>
+            <column name="verdict_message" type="LONGTEXT"/>
+            <column name="verdict_point_deduction" type="INT"/>
+        </createTable>
+        <addColumn tableName="post">
+            <column name="plagiarism_case_id" type="BIGINT"/>
```

Comment:
2. in the context of 1 (above) this does not make sense. While I would recommend in general to only use a unidirectional association from plagiarism_case to post, in any case you would need to use mappedBy here